schemas/test/en/test_section_summary.json

```diff
@@ -135,6 +135,38 @@
                             ]
                         }
                     ]
+                },
+                {
+                    "id": "address-listed",
```

Comment:
This has been added so there is now a block between the new question that would appear on a routing change and the section summary. With the new question being last in the section, it could have been routing normally to the section summary rather than using the return_to and as such failing